src/js/monkey.js

```diff
@@ -78,7 +78,7 @@ window.Monkey = module.exports = (function () {
     }
 
     promise = promise
-      .then(Monkey._generateUrls(options.preload))
+      .then(Monkey._generateUrls(options.preload, options))
```

Comment:
Yes, but now you have the `options` object so you can just replace `preload` with `options.preload` :)